frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/admin/TodoPortlet.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Siemens AG, 2013-2015. Part of the SW360 Portal Project.
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.sw360.portal.portlets.admin;
+
+import org.apache.log4j.Logger;
+import org.apache.thrift.TException;
+import org.eclipse.sw360.datahandler.permissions.PermissionUtils;
+import org.eclipse.sw360.datahandler.thrift.RequestStatus;
+import org.eclipse.sw360.datahandler.thrift.licenses.LicenseService;
+import org.eclipse.sw360.datahandler.thrift.licenses.Todo;
+import org.eclipse.sw360.datahandler.thrift.users.User;
+import org.eclipse.sw360.datahandler.thrift.users.UserGroup;
+import org.eclipse.sw360.portal.common.UsedAsLiferayAction;
+import org.eclipse.sw360.portal.portlets.Sw360Portlet;
+import org.eclipse.sw360.portal.portlets.components.ComponentPortletUtils;
+import org.eclipse.sw360.portal.users.UserCacheHolder;
+
+import javax.portlet.*;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+import static org.eclipse.sw360.portal.common.PortalConstants.*;
+
+/**
+ * Todo portlet implementation
+ *
+ * @author nikolas@endocode.com
+ */
+public class TodoPortlet extends Sw360Portlet {
+
+    private static final Logger log = Logger.getLogger(TodoPortlet.class);
+
+
+    //! Serve resource and helpers
+    @Override
+    public void serveResource(ResourceRequest request, ResourceResponse response) {
+
+        final String id = request.getParameter("id");
```

Comment:
Isn't there a static var for that?

Reply:
I don't know. Please provide some pointers on where i can find this static var.

Reply:
It seems like there is none yet but normally the available parameters are defined in `org.eclipse.sw360.portal.common.PortalConstants`. Maybe just the new ones for Todos there as well?